app/models/miq_server/worker_management/monitor/quiesce.rb

```diff
@@ -36,7 +36,14 @@ def quiesce_workers_loop
     worker_monitor_poll = (@worker_monitor_settings[:poll] || 1.seconds).to_i_with_method
 
     miq_workers.each do |w|
-      MiqEnvironment::Command.is_podified? && w.containerized_worker? ? w.delete_container_objects : stop_worker(w)
+      if MiqEnvironment::Command.is_podified? && w.containerized_worker?
+        w.delete_container_objects
+      elsif w.systemd_worker?
+        w.stop_systemd_unit
+        w.disable_systemd_unit
```

Comment:
These two seems like a pair (and is repeated elsewhere), so would it make sense to have a method that encapsulates these 2 operations?

Reply:
Ah yeah, I see `start_runner_via_systemd` does the opposite of this anyway, so it feels there should be a `stop_runner_via_systemd` that is these 2 methods

Reply:
:+1: done